src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java

```diff
@@ -166,6 +183,29 @@ public CassandraConnection(final SessionHolder sessionHolder) throws SQLExceptio
         });
     }
 
+    public CassandraConnection(Session cSession, String currentKeyspace, ConsistencyLevel defaultConsistencyLevel, boolean debugMode) {
```

Comment:
To be compliant with our Checkstyle rules, please mark each parameter as final:
```
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:186:32: Parameter cSession should be final. [FinalParameters]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:186:50: Parameter currentKeyspace should be final. [FinalParameters]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:186:74: Parameter defaultConsistencyLevel should be final. [FinalParameters]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:186:116: Parameter debugMode should be final. [FinalParameters]
```
(you can find this in the build logs)